libcloud/test/compute/fixtures/cloudscale/list_images.json

```diff
@@ -0,0 +1 @@
+[{"slug":"ubuntu-16.10","name":"Ubuntu 16.10 (2016-10-17)","operating_system": "Ubuntu"}]
```

Comment:
I have actually removed the pretty printing because other drivers also don't use it (and this one is pretty short).